videostore/videostore.go

```diff
@@ -278,9 +317,19 @@ func NewRTPVideoStore(config Config, logger logging.Logger) (RTPVideoStore, erro
 		logger:       logger,
 		config:       config,
 		workers:      utils.NewBackgroundStoppableWorkers(),
+		renameWorker: utils.NewBackgroundStoppableWorkers(),
 	}
 
 	vs.workers.Add(vs.deleter)
+
+	if renamer != nil {
+		vs.renameWorker.Add(func(ctx context.Context) {
+			if err := renamer.processSegments(ctx); err != nil {
+				vs.logger.Errorf("failed to process segments: %v", err)
+			}
+		})
```

Comment:
processSegments never returns an error:
```suggestion
		vs.renameWorker.Add(renamer.processSegments)
```